Add 0xB10C, Vincenzo, Niklas, Bruno (#78)
bitcoinbrink/website: _posts/2022-04-04-grant-announcements.md

---
title: "Announcing Brink's Latest Round of Developer Grants"
permalink: /blog/2022/04/04/grantees/
layout: post
author: brink
name: "brink"
alt: Brink announces latest round of developer grants
category: "News"
description: Brink announces the latest round of developer grants.
---

Brink is proud to announce the recipients of its latest round of open source developer grants.
After an open grant application process, the [Brink board of directors][brink
board] has selected four developers who have demonstrated continued commitment to

> open source Bitcoin development. The grant recipients are:

Reviewer: Nitpick: I think the repeated "X writes of his Bitcoin experience and goals" can be expressed universally here, e.g. change "The grant recipients are:" to "Each of the four grant recipients describes their Bitcoin experience and goals below:"

## "0xB10C"

<img src="/assets/images/0xb10c.png" alt="Picture of 0xb10c" style="float:left;border:1px solid
darkgray;margin-right:25px;width:150px;" />

0xB10C writes of his Bitcoin experience and goals:

I am a Bitcoin developer curious about what is happening on the Bitcoin

> network. Over the past years, I've built tools to visualize and observe, for

Reviewer: Nit: the "for example" can be dropped from this sentence to simplify it.  (The example part is especially unneeded when the sentence ends with "and other protocol level statistics".)

as a freelance developer for ShiftCrypto and CoinMetrics. At CoinMetrics, I
helped bootstrap [Farum][farum], a real-time network and attack monitoring suite for
Bitcoin. My work includes patching Bitcoin Core to expose a real-time interface
for changes to the node's mempool and chain view, along with software to monitor
the mining jobs of hundreds of Stratum servers to detect, for example, network
splits early.

I decided to focus entirely on open source projects in early 2021 and accepted
a one-year grant offer from Coinbase. During this year, I built
[miningpool-observer][miningpool-observer], mentored two [Summer of Bitcoin][sob] mentees, introduced
tracepoints for [Userspace, Statically Defined Tracing to Bitcoin Core][usdt], started
working on making regular reorgs on the Signet test network a reality, and
published an open-content blog for Bitcoin developers called
[bitcoin-dev.blog][bitcoin dev blog].

With support from Brink in 2022, I plan to continue work on tracepoints in
Bitcoin Core, including interface tests, review, and the addition of further
tracepoints. Another goal is to launch a Signet with regular reorgs and a tool

> to explore reorgs visually. After flooding attacks on the Bitcoin P2P network in

Reviewer: ```suggestion
to explore reorgs visually. After observing flooding attacks on the Bitcoin P2P network in
```

I found this sentence ambiguous---it sounded like 0xB10C *caused* the flooding attacks.  Hopefully this change clarifies that he merely observed them.

detection tool in 2022. The goal is to feed the observations into Bitcoin and
Bitcoin Core development, improving the P2P network robustness. I also plan to
write up further Mempool Observations and start a similar series of blog posts
for P2P network observations.

[@0xB10C][0xB10C twitter]

## Vincenzo Palazzo

<img src="/assets/images/vincenzo-palazzo.jpg" alt="Picture of Vincenzo Palazzo" style="float:left;border:1px solid
darkgray;margin-right:25px;width:150px;" />

Vincenzo writes of his Bitcoin experience and goals:

I am an open source software developer and in my free time a Master’s
Student at the University of Pisa in Italy. I became interested in Bitcoin in
2019 due to my bachelor’s degree. In the year after discovering Bitcoin, I
became involved in different projects related to the Lightning Network protocol.

I have been working on the C-Lightning implementation as an active
contributor for the last nine months. I am the maintainer of
[lnprototest][github lnprotest], a Lightning BOLT Protocol Test Framework,
and maintainer of [rust-clightning-rpc][github rust cl rpc], an interface from
rust to the C-Lightning daemon through RPC.

> With my Brink grant, I will focus to improve and support the C-Lightning

Reviewer: ```suggestion
With my Brink grant, I will focus on improving and supporting the C-Lightning
```

Nit

implementation with new features as well as review pull requests. I also plan
to contribute to review of the Lightning specification.

[@palazzovincenzo][vincenzo twitter]

## Niklas Gögge

<img src="/assets/images/niklas-goegge.jpg" alt="Picture of Niklas Gögge" style="float:left;border:1px solid
darkgray;margin-right:25px;width:150px;" />

Niklas writes of his Bitcoin experience and goals:

I first got interested in Bitcoin in 2017, but it wasn’t until late 2018 when I
read Mastering Bitcoin that I got fully obsessed with the technical side of Bitcoin.

With the start of the pandemic I had a lot of time on my hands and

> start contributing to Bitcoin. I found [multiple talks][tadge talks] of Tadge Dryja talking

Reviewer 2: ```suggestion
started contributing to Bitcoin. I found [multiple talks][tadge talks] of Tadge Dryja talking
```

Utreexo and propose a proof-of-concept PR that people can test and review. In
addition to working on Utreexo I plan on making more security focused
contributions to Bitcoin Core and review PRs.

[@dergoegge][niklas twitter]

## Bruno Garcia

<img src="/assets/images/bruno-garcia.jpg" alt="Picture of Bruno Garcia" style="float:left;border:1px solid
darkgray;margin-right:25px;width:150px;" />

Bruno writes of his Bitcoin experience and goals:

I discovered Bitcoin in 2015 and fell in love with the concept of digital money
and how this could improve and collaborate for freedom. Years later, I started
studying the Bitcoin protocol more, reading books like Programming Bitcoin,
Mastering Bitcoin, and other materials from the internet.

In 2021, I attended the Chaincode [Bitcoin Protocol Development
Seminar][chaincode learning] where I learned more about the
Bitcoin protocol and started contributing to Bitcoin Core. Since I have a DevSec
and QA background, my journey started by reviewing and reading the [functional
tests][bitcoin functional tests]. This is how I discovered an
inconsistency in the taproot test and got my first PR merged ([#21081][bitcoin 21081]).

> Since then, I haven’t stopped contributing to Bitcoin Core by testing and

Reviewer: Nit: s/haven't stopped/have continued/

(In English, it's preferred to use the positive form of a phrase rather than the negative.)

reviewing PRs, improving the
test coverage, and recently P2P and wallet work. I also wrote a coin selection
library in Typescript that implements the same strategies as Bitcoin Core. In

> addition to this develment and review, I mentor some students from one of the best

Reviewer 2: ```suggestion
addition to this development and review, I mentor some students from one of the best
```
